### PR TITLE
curl, bump version

### PR DESCRIPTION
**Pull request by Begasus**

### CODE REVIEW

File: net-misc/curl/curl-8.3.0.recipe [hunk 1]
```diff
@@ -73,7 +73,7 @@ TEST_REQUIRES="
 	cmd:id
 	cmd:perl
 	cmd:python3
-	cmd:sftp_server
+#	cmd:sftp_server
```
[the remark this answers is not in the document]
Comment (korli) [reply net-misc/curl/curl-8.3.0.recipe:76]: removed in https://github.com/haikuports/haikuports/commit/15380ca2d96173a39a1cc1ac38153c248bce2d20

[the remark this answers is not in the document]
Comment (korli) [reply net-misc/curl/curl-8.3.0.recipe:76]: actually sftp-server might still be in the openssh hpkg under $lib/ssh

**Comment by Begasus** [comment]
Don't merge, I think this caused strange behaviour on WebPositive (weird layout/not showing pages ...)

**Comment by Begasus** [comment]
Closing for now untill it's behaving nicely. :)

**Pull request closed without merging**